packages/vite/src/node/__tests__/config.spec.ts

```diff
@@ -266,3 +270,19 @@ describe('preview config', () => {
     })
   })
 })
+
+describe('loadConfigFromFile', () => {
+  const root = resolve(__dirname, './packages/config')
+  const configEnv: ConfigEnv = { command: 'serve', mode: 'development' }
+  test.each([
+    'vite.config.c.js',
+    'vite.config.m.js',
+    'vite.config.cjs',
+    // 'vite.config.mjs',
```

Comment:
Dynamic import will trigger the `segmentation fault` error. I don't know much about it. It would be nice if someone could help me